src/index.js

```diff
@@ -74,7 +74,10 @@ class MongoSlowQueryChecker {
       const promise = deferred();
       this.db.runCommand(command, promise.defer());
       ops = await promise;
+    } else if (typeof this.db.adminCommand === 'function') {
+      ops = await this.db.adminCommand(command);
     } else {
+      // Assume this.db is a reference to the 'admin' db
```

Comment:
Can/should we check? Will this fail otherwise?

Reply:
It'll fail with a useful error message ('currentOp may only be run against the admin database.'); I didn't think it was worth checking ourselves.

Reply:
Perfect.